waku/standards/core/17/rln-relay.md

```diff
@@ -2,7 +2,7 @@
 slug: 17
 title: 17/WAKU2-RLN-RELAY
 name: Waku v2 RLN Relay
-status: draft
+status: stable
```

Comment:
The changes to the draft spec below are useful. However, I wouldn't move this to stable yet. As @rymnc mentions, we are currently transitioning to RLNv2 and will stabilize on that version. I think we also need to explain the role of gossipsub scoring in our protocol before it would be a complete description.